### PR TITLE
DEV-4516-dev-ops-institute-content

### PR DESCRIPTION
**Pull request by zamazzal**
> Before starting, please choose the relevant pull request **Labels**, **Reviewers**, and **Assignees**

### Why?
Adds a new subject called "Road-To-DOFD" to prepare students for the DevOps Foundation(DOFD) Certification exam from the DevOps Institute.

### Solution Overview
The solution involves creating a new subject and audit that provides an overview of the certification, exam domains, study materials, and exam preparation tips in order to prepare the students for the DOFD Certification.

### Implementation Details
A new subject and audit were created with structured content covering all the necessary information for exam preparation.

### CODE REVIEW

File: subjects/devops/road-to-dofd/audit/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+###### Was the student familiar with the DevOps Institute DevOps Foundation Exam Guide?
+
+###### Did the student read relevant books, articles, and whitepapers on DevOps?
+
+###### Did the student show proficiency in the domains covered in the exam guide?
+
+##### Check learner knowledge:
+
+###### Did the student demonstrate a clear understanding of DevOps principles and concepts?
```
[the remark this answers is not in the document]
Comment (zamazzal) [reply subjects/devops/road-to-dofd/audit/README.md:15]: fixed

File: subjects/devops/road-to-dofd/audit/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+###### Was the student able to identify common automation and tooling used in DevOps?
+
+###### Did the student understand the significance of measurement and metrics in DevOps?
+
+###### Was the student able to explain the importance of sharing and communication in a DevOps environment?
+
+###### Was the student able to provide examples of real-world implementations of DevOps practices?
+
+##### Choose questions from the Sample Examination 1 and 2 sections in the Exam guide:
```
[the remark this answers is not in the document]
Comment (zamazzal) [reply subjects/devops/road-to-dofd/audit/README.md:31]: fixed

File: subjects/devops/road-to-dofd/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+5. Measurement and Metrics
+6. Sharing and Communication
+
+### Study Materials
+
+You should refer to the following study materials to prepare for the exam:
+
+- DevOps Institute DevOps Foundation Exam Guide English version: [DOFD-English-Exam-Study-Guide.pdf](./resources/DOFD-English-Exam-Study-Guide.pdf)
+- DevOps Institute DevOps Foundation Exam Guide French version: [DOFD-French-Exam-Study-Guide.pdf](./resources/DOFD-French-Exam-Study-Guide.pdf)
+- DevOps Institute DevOps Foundation Exam Guide Spanish version: [DOFD-Spanish-Exam-Study-Guide.pdf](./resources/DOFD-Spanish-Exam-Study-Guide.pdf)
```
[the remark this answers is not in the document]
Comment (zamazzal) [reply subjects/devops/road-to-dofd/README.md:32]: done

File: subjects/devops/road-to-dofd/README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+4. Automation and Tooling
+5. Measurement and Metrics
+6. Sharing and Communication
+
+### Study Materials
+
+You should refer to the following study materials to prepare for the exam:
+
+- DevOps Institute DevOps Foundation Exam Guide English version: [DOFD-English-Exam-Study-Guide.pdf](./resources/DOFD-English-Exam-Study-Guide.pdf)
+- DevOps Institute DevOps Foundation Exam Guide French version: [DOFD-French-Exam-Study-Guide.pdf](./resources/DOFD-French-Exam-Study-Guide.pdf)
```
[the remark this answers is not in the document]
Comment (zamazzal) [reply subjects/devops/road-to-dofd/README.md:31]: done

File: subjects/devops/road-to-dofd/README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+3. Processes and Practices
+4. Automation and Tooling
+5. Measurement and Metrics
+6. Sharing and Communication
+
+### Study Materials
+
+You should refer to the following study materials to prepare for the exam:
+
+- DevOps Institute DevOps Foundation Exam Guide English version: [DOFD-English-Exam-Study-Guide.pdf](./resources/DOFD-English-Exam-Study-Guide.pdf)
```
[the remark this answers is not in the document]
Comment (zamazzal) [reply subjects/devops/road-to-dofd/README.md:30]: fixed

File: subjects/devops/road-to-dofd/README.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
+## Road-To-DOFD
+
+<center>
+<img src="./resources/dofd-logo.png?raw=true" style = "width: 400px !important; height: 400px !important;"/>
```
[the remark this answers is not in the document]
Comment (zamazzal) [reply subjects/devops/road-to-dofd/README.md:4]: image removed

File: subjects/devops/road-to-dofd/audit/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (davhojt) [reply subjects/devops/road-to-dofd/audit/README.md]: There were some others, I resolved them here: https://github.com/01-edu/public/pull/2030/commits/1a11fdbc527607080dc9de220a4896a9ccca8a93

**Review by davhojt — APPROVED** [review]
Yo @zamazzal.

I made two commits, you can check them before merge.

**Comment by zamazzal** [comment]
Done checked and merged

**Pull request merged**